emhttp/plugins/dynamix/styles/jquery.sweetalert.css

```
@@ -1,219 +1,1573 @@
+  }
+
+  .sa-error-container {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    gap: 1rem;
+    overflow: hidden;
+    padding: 0 1rem;
+    max-height: 0;
+    webkit-transition: padding 0.15s, max-height 0.15s;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)